cost/downsize_instance/downsize-instance-add-tags.pt

```diff
@@ -0,0 +1,114 @@
+name "Downsize Instances Add Tags Policy Template"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that checks cooldown time and adds back in time"
+long_description "Version: 1.0"
+severity "medium"
+category "Cost"
+
+permission "perm_instance_and_monitoring_metrics" do
+  label "Permissions for instances and monitoring metrics"
+  actions "rs_cm.index", "rs_cm.show", "rs_cm.data"
+  resources "rs_cm.instances", "rs_cm.monitoring_metrics"
+end
```

Comment:
I'm surprised it works. I'd separate the resources `rs_cm.instances` and `rs_cm.monitoring_metrics` as `rs_cm.instances` doesn't really have the `rs_cm.data` action.